Yubico.YubiKey/docs/users-manual/application-piv/apdu/change-ref.md

```diff
@@ -25,10 +25,14 @@ Which element to change is given in the P2 field of the APDU and the current and
 data (old and new PIN or PUK) are given in the data field. The data is simply the two
 values concatenated.
 
-Both the PIN and PUK are allowed to be 6 to 8 characters, but if one is less than 8, it
-will be padded with 0xff. For example, the default PIN is "123456", but on the device,
+Both the PIN and the PUK are allowed to be 6 to 8 characters. If one is less than 8, it
```

Comment:
@equijano21 the PUK has to be 8 characters and 8 characters only. The PIN can be 6-8 characters and will be padded if less than 8.